content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -84,20 +84,11 @@ If you have a `.github/dependabot.yml`, add:
     interval: monthly
 ----
 
-If you did not yet have such a file, create it now. For example:
+If you did not yet have such a file, create it now:
 
-[source,yaml]
+[source,shell]
 ----
-version: 2
-updates:
-- package-ecosystem: maven
-  directory: /
-  schedule:
-    interval: monthly
-- package-ecosystem: github-actions
-  directory: /
-  schedule:
-    interval: monthly
+curl -so .github/dependabot.yml https://raw.githubusercontent.com/jenkinsci/archetypes/master/common-files/.github/dependabot.yml
```

Comment:
Post review, nitpicking: using long flags is a great help for a lot of people (when the long flags are available) to understand what a command is doing:

```shell
curl --silent --output .github/dependabot.yml https://raw.githubusercontent.com/jenkinsci/archetypes/master/common-files/.github/dependabot.yml
```

Also, adding the `-L/--location` flag would make this example ready for the future (case of renaming on our side for instance):

```shell
curl --silent --location --output .github/dependabot.yml https://raw.githubusercontent.com/jenkinsci/archetypes/master/common-files/.github/dependabot.yml
```

Reply:
This is consistent with the rest of the file, feel free to file a PR to address existing issues :)

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/5677